doc/release-notes.md

```diff
@@ -29,7 +29,7 @@ using version < 0.13 you will have to reindex (start with -reindex-chainstate
 or -reindex) to make sure your wallet has all the new data synced. Upgrading
 from version 0.13 should not require any additional actions.
 
-When upgrading from a version prior to 18.0.0, the
+When upgrading from a version prior to 18.0.1, the
```

Comment:
not technically true

Reply:
yeah, but 18.0.0 was never released so it's ok-ish imo :)